### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Nooshu**
Minimal change to the README.md. Currently, it mentions WCAG 2.1 but when you click on the link it goes to a WCAG 2.2 page.

### CODE REVIEW

**Review by 36degrees — APPROVED** [review]
Good spot, thanks @Nooshu! This just needs a second review from the team and then we can get it merged.

**Review by romaricpascal — APPROVED** [review]

**Pull request merged**
